src/layman/rest_multipublication_test.py

```diff
@@ -191,6 +191,14 @@ def assert_response(response, expected_publications, expected_headers):
             'X-Total-Count': '5',
             'Content-Range': 'items 1-5/5'
         },),
+        (authn_headers_user2, {'full_text_filter': 'TOUCK'}, [
```

Comment:
I would like also to have test that results matching by `@@` operator are ordered before results matching by `LIKE` operator.